### ISSUE TITLE
Embed screenshots in report

### ISSUE THREAD

**Issue by subratkumarmandal** [uttesh/pdfngreport#11]
Is there a way to embed screenshots in report or provide link ?

In my case I take screenshots on test case failure .

What's the way to configure this option ?

**Comment by uttesh** [comment]
@subratkumarmandal Yes its under development, lot of request on that integration, once done i will notify thanks,

**Comment by uttesh** [comment]
It already there in the library. set the following in the pdfngreport.properties file

Below setting only for selenium user for selenium failed screen shot link show related
pdfreport.show.selenium.screenshot.link=show
pdfreport.selenium.failed.test.screenshot.outputdir=path to screenshots folder

Please follow below standards to name your screen shot image file name and type from code/manual
image type : png
image name : className + "_" + methodName

sample: 
class: sampleTest.java
method name: sumTest

image name: sampleTest_sumTest.png
